source/mainnet/net/resources/sirius-testnet-reset-nodes.rst

```diff
@@ -0,0 +1,62 @@
+.. _sirius-testnet-reset-nodes:
+
+==================================
+Node migration after Testnet reset
+==================================
+
+The following contains information about how to migrate your node after Testnet reset depending on your platform.
+
+Windows
+=======
+
+To update the Windows node you need to delete the existing testnet database and
+install and run the new node. The steps are:
+
+1. Stop the existing testnet node following :ref:`instructions
+   <install-upgrade-node-windows>` on how to upgrade a node.
+2. Navigate to the testnet node's data directory. This was selected during node
+   installation. By default it will be ``<CommonAppDataFolder>\Concordium\Node
+   Runner\testnet\data`` where ``<CommonAppDataFolder>`` is usually
+   ``C:\ProgramData``.
+3. Delete the directory named ``database-v4``.
```

Comment:
This didn't work. I had to delete everything in data folder before it actually worked. I am not sure why I was getting an error message that Genesis is wrong. Maybe installer didn't overwrite old Genesis.dat.

Reply:
I updated the instructions to delete the testnet\data folder.